README.md

```diff
@@ -72,6 +64,19 @@ The event subscription should be set to `localhost` to reduce network traffic.
 Please refer to options section for more.
 
 
+## `Marathon Labels`
+
+`AppCop` is using `Marathon` labels to communicate actions or to tune execution logic.
+
+Used labels:
+
+Name                      |       Possible values     |    r/w   |    Description
+--------------------------|---------------------------|----------|------------------
+appcop                    | `suspend`, `scaleDown`    |    w     | Every time `AppCop` scales or suspend application, put appropriate label in app definition
+APP_IMMUNITY              |   `false`, `true`         |    r     | Application with this value is excused from all criminal actions on cluster. Use this feature wisely, because used to exensibly negates whole purpose of AppCop.
```

Comment:
Application with this value is excused from all criminal actions on cluster. Use this feature wisely, because used to exensibly negates whole purpose of AppCop. 
->
When AppCop encounters this label in app definition, treats it as immune to all penalties (excused from all criminal actions on cluster). Use this feature wisely, because if used to often it could defeat whole purpose for using AppCop.